modules/tiles/src/tileset/tileset-3d.ts

```diff
@@ -103,10 +103,10 @@ const DEFAULT_PROPS: Props = {
   maximumMemoryUsage: 32,
 
   // Indicates this a tile's content was loaded
-  onTileLoad: (tile) => {},
+  onTileLoad: () => {},
   // Indicates this a tile's content was unloaded
-  onTileUnload: (tile) => {},
-  onTileError: (tile, message, url) => {},
+  onTileUnload: () => {},
```

Comment:
This looks like we are taking something away... can we fix types instead of remove them?